src/apps/backend/modules/share-tasks/types.ts

```diff
@@ -0,0 +1,39 @@
+//backend\modules\share-tasks\types.ts
+import { ApplicationError } from '../application';
+import { HttpStatusCodes } from '../http';
+
+// Define the SharedTask class
+export class SharedTask {
+  id: string;
+  task: string;
+  account: string;
+  title: string;
+  description: string;
+  username: string;
+}
```

Comment:
title and description should be part of the task and the username should be part of the account